contrib/win32/win32compat/misc.c

```diff
@@ -831,13 +834,17 @@ w32_stat(const char *input_path, struct w32_stat *buf)
 	return fileio_stat(resolved_path(input_path), (struct _stat64*)buf);
 }
 
```

Comment:
Since you already made changes to symbolic links, it would be great if you can implement w32_symlink(), link() implementation.

Reply:
I can probably look into those, but I'd like to get this pull in first before I work on those.